CHANGELOG.md

```diff
@@ -13,16 +13,16 @@ PLEASE REMEMBER TO CHANGE THE '..master' WITH AN ACTUAL TAG in GITHUB LINK.
 * New features:
   * The {func}`jax2tf.convert` now has support for `pjit` and `sharded_jit`.
 
+  * The {func}`jax2tf.convert` generates custom attributes with location information
+   in TF ops. The code that XLA generates after jax2tf
+   has the same location information as JAX/XLA.
+
 * Breaking changes:
 
 * Bug fixes:
   * The {func}`jax2tf.convert` now scopes the `enable_xla` conversion parameter
     properly to apply only during the just-in-time conversion
     ({jax-issue}`#6720`).
-  * Fixed assertion failure in {func}`jax2tf.call_tf` when used with captured
```

Comment:
Accidentally dropped?

Reply:
Actually, it was a duplicate. This was already includes in 0.2.13